### ISSUE TITLE
A few ideas/questions

### ISSUE THREAD

**Issue by DarkObscurity** [itflow-org/itflow#201]
Hey!

I love the product and have been looking around  in it for a little and a few questions/ideas came to mind!

I have been working quite a lot in IT-Glue in the past and for my own purpose I was looking for an open source alternative and this is the only project so far that comes close to it! So I'm really grateful for that!

I did have some questions regarding the documentation part.
IT-Glue is all about documentation and having the tools to do so in a easy to access way>

When I was looking around in the asset documentation I noticed that the way this is documented is in tiny pop-up windows instead of documentation pages with things like related items and I was  curious if this might change in the future or of this is how it's going to stay?

I personally value proper documentation a lot with the ability to create guides on the places where people expect them (For example when opening a specific server asset, it would be the perfect place to have any guides/special needs or manuals located in a full page that opens.) 

I also saw that the location feature is limited to the US, and I understand that it might be a lot to add to have the whole world in there, but maybe it might be an idea to allow people to just type in the information them self for the time being? I don't know if this will break anything though...

And are there any plans for adding custom side-bar layouts with page lay-outs? IT-Glue has a really nice system with a sort of customer lay-outs where for example you could create documentation sections for things like Microsoft 365. These pages then contain certain fields/drop down lists/options to be filled in so one can, for each customer, fill in for example what kind of Microsoft 365 services are being used and the lay-out can be distributed across all customers so a baseline can be set.

Long story short, I am curious if the documentation side will be extended in the future! :)

**Comment by johnnyq** [comment]
Thank you for the kind words @DarkObscurity 

1. Yeah I agree, the tiny modal window for documentation is quite limited. I will eventually make this full screen and allow folder, subfolder categorization of documents along with PDF exportation some time in the very near future.
2. I think that would be great for assets,in fact we may be able to make an asset documentation relationship to the documents section. This will also be done
3. We have recently added full country support see 72cf61ab61c3b91324ea71b9795cf978abdf6ff0
4. I have never actually used ITGlue and have only seen some screenshots so im not fully aware of this feature, but it sounds neat and would love to implement something similar.

Thank you for your ideas and kind words, keep them flowing.

**Comment by DarkObscurity** [comment]
> Thank you for the kind words @DarkObscurity
> 
> 1. Yeah I agree, the tiny modal window for documentation is quite limited. I will eventually make this full screen and allow folder, subfolder categorization of documents along with PDF exportation some time in the very near future.
> 2. I think that would be great for assets,in fact we may be able to make an asset documentation relationship to the documents section. This will also be done
> 3. We have recently added full country support see [72cf61a](https://github.com/johnnyq/itflow/commit/72cf61ab61c3b91324ea71b9795cf978abdf6ff0)
> 4. I have never actually used ITGlue and have only seen some screenshots so im not fully aware of this feature, but it sounds neat and would love to implement something similar.
> 
> Thank you for your ideas and kind words, keep them flowing.

Really nice to hear! If you have something in BETA I would love to test it for you.
My coding knowledge is quite limited, but I'm an IT Infrastructure/Cloud Engineer so working with servers and stuff is my thing! 

I actually have access to IT-Glue, so if you want I could give you a lot of info if you'd like! 

It would be amazing to have the documentation capabilities of IT-Glue in an open source self hosted environment!

**Comment by johnnyq** [comment]
hi @DarkObscurity I wanted to send you a PM and I don't have your contact info. Send me a message on our contact form so I can get in touch with you https://pittpc.com

**Comment by DarkObscurity** [comment]
> hi @DarkObscurity I wanted to send you a PM and I don't have your contact info. Send me a message on our contact form so I can get in touch with you https://pittpc.com

I've just send a message through the contact form!

**Comment by DarkObscurity** [comment]
Hello @johnnyq 

I filled in the contact form but haven't yet heard anything from you, did you get my details?
